python-sdk/langwatch/examples/langgraph_rag_bot_with_threads.py

```
@@ -0,0 +1,330 @@
+import datetime
+import os.path
+import pickle
+from typing import Dict, List, Literal, Optional
+from dotenv import load_dotenv
+
+from langwatch.types import RAGChunk
+
+load_dotenv()
+
+import chainlit as cl
+from langchain.prompts import ChatPromptTemplate
```

Ruff (0.8.2)

`langchain.prompts.ChatPromptTemplate` imported but unused

Remove unused import: `langchain.prompts.ChatPromptTemplate`

(F401)